src/components/LogosSection/TrustedBy/index.js

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { FormattedMessage } from "gatsby-plugin-intl"
+import { useStaticQuery, graphql } from "gatsby"
+import Img from "gatsby-image"
+
+import { trustedByImage, andMore } from "./style"
+
+const TrustedBy = () => {
+  const Image = ({ fluid }) => <Img css={trustedByImage} fluid={fluid} />
```

Comment:
Please put this outside of `<BackedBy />`

Reply:
done.